pkg/server/commands/reverseexpand/reverse_expand.go

```diff
@@ -432,6 +432,9 @@ func (c *ReverseExpandQuery) readTuplesAndExecute(
 	)
 	defer filteredIter.Stop()
 
+	ctx, span := tracer.Start(ctx, "readTuplesAndExecute")
```

Comment:
What's the rationale for putting this here as opposed to at the top of the function body? It's more standard to put this at the top of the function body, and this is what we've done elsewhere as well.

If there's a rationale reason, it should be commented in the code to indicate as such.

Reply:
Good point 😆  no reason, updated